test_framework/scripts/longhorn-setup.sh

```
@@ -2,30 +2,21 @@
-# Longhorn version tag (e.g v1.1.0), use "master" for latest stable
-# we will use this version as the base for upgrade
-LONGHORN_STABLE_VERSION=${LONGHORN_STABLE_VERSION:-master}
-LONGHORN_STABLE_MANIFEST_URL="https://raw.githubusercontent.com/longhorn/longhorn/${LONGHORN_STABLE_VERSION}/deploy/longhorn.yaml"
-
-# for install Longhorn by manifest
-LONGHORN_MANIFEST_URL="https://raw.githubusercontent.com/longhorn/longhorn/${LONGHORN_INSTALL_VERSION}/deploy/longhorn.yaml"
 
-# for install Longhorn by helm chart
-LONGHORN_REPO_URL="https://github.com/longhorn/longhorn"
-LONGHORN_REPO_DIR="${TMPDIR}/longhorn"
+LONGHORN_INSTALL_METHOD="manifest"
```

Shellcheck (0.10.0)

[warning] LONGHORN_INSTALL_METHOD appears unused. Verify use (or export if used externally).

(SC2034)